docs/releases/3.1.1.txt

```diff
@@ -25,3 +25,6 @@ Bugfixes
   :meth:`~django.db.models.query.QuerySet.select_for_update()`. When using
   related fields pointing to a proxy model in the ``of`` argument, the
   corresponding model was not locked (:ticket:`31866`).
+
+* Fixed type inspection of deprecated keyword arguments in
+  ``TemplateView.get_context_data()`` (:ticket:`13297`).
```

Comment:
13297 -> 31877

Reply:
Woops, PR number instead of ticket 😅 I was clearly too tired when I tried this.